support/cas-server-support-aws-cognito-authentication/src/main/java/org/apereo/cas/config/AmazonCognitoAuthenticationConfiguration.java

```diff
@@ -89,6 +91,14 @@ public AuthenticationHandler amazonCognitoAuthenticationHandler() {
             amazonCognitoIdentityProvider(),
             cognito,
             amazonCognitoAuthenticationJwtProcessor());
+
+        val mappedAttributes = cognito.getMappedAttributes();
```

Comment:
Remove, and do the transformation inside the handler.

Reply:
Same as above. I will remove it. I see that it is lot cleaner to do the transformation inside the handler.